migrations/test-only-migrations/R__Test-Data-Creation.sql

```diff
@@ -454,6 +456,10 @@ INSERT INTO public.officer
 (officer_guid, user_id, create_user_id, create_utc_timestamp, update_user_id, update_utc_timestamp, person_guid, office_guid)
 VALUES('623fbd02-dff5-4d45-9ab3-f0acbe8d83ed'::uuid, 'JXDUNSDO', 'FLYWAY', '2024-01-22 22:16:16.754', 'FLYWAY', '2024-01-22 22:20:48.186', '17439e37-ca97-4031-a009-b23ca58c21e1'::uuid, '5128179c-f622-499b-b8e5-b39199081f22'::uuid)
 ON CONFLICT DO NOTHING;
+INSERT INTO public.officer
```

Comment:
Due to the order the scripts get run the two insert statements you've added won't actually run as the prod versions ran first, however it's probably good to include them.

What you might want though is an update statement that updates your office_guid to 4a5a94b1-bd47-4611-a577-861d97089903  as this will move you OUT of COSHQ and into something more usable (See lines circa. 470-480)

Reply:
Updated 👍